### PR TITLE
Changes to make screens more responsive + incident status toggler change

### LINKED ISSUE #1000: Nav: Few pages does not fit Mac's 13" with the new nav
Reproduced on pages:
- Alert list
- Specific schedule edit

https://user-images.githubusercontent.com/3284841/207865654-78155277-8cd1-48be-9c8d-8f1c4dcf732f.mov

P.S. I was not able to reproduce it in the developer mode in chrome 0_o

### PR DESCRIPTION
**Pull request by teodosii**
# What this PR does

- Changes to make a few screens be more responsive
- Removed incident actions and replaced incident status with a toggler
- Renamed `IncidentStatus.new` to `IncidentStatus.Firing`
- Removed old schedules code (unused)

## Which issue(s) this PR fixes

#1000 

## Checklist

- [x] `CHANGELOG.md` updated

### CODE REVIEW

File: grafana-plugin/src/components/SchedulesFilters/SchedulesFilters.tsx [hunk 1]
```diff
@@ -1,59 +1,94 @@
...
   value: SchedulesFiltersType;
   onChange: (filters: SchedulesFiltersType) => void;
-  className?: string;
 }
 
-const SchedulesFilters = ({ value, onChange, className }: SchedulesFiltersProps) => {
-  const handleDateChange = useCallback(
-    (date: Date) => {
-      onChange({ selectedDate: moment(date).format('YYYY-MM-DD') });
+const SchedulesFilters = (props: SchedulesFiltersProps) => {
```
Comment (teodosii) [inline grafana-plugin/src/components/SchedulesFilters/SchedulesFilters.tsx:18]: Hmm, I haven't touched these parts. Don't know why these changes show up in here

File: grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import { Tooltip } from '@grafana/ui';
+import { debounce } from 'throttle-debounce';
+
+interface MatchMediaTooltipProps {
```
Comment (maskin25) [inline grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx:6]: you can pick props from `grafana/ui` `TooltipProps` like
` type MatchMediaTooltipProps = Pick<TooltipProps, 'placement'|'content'> `

File: grafana-plugin/src/containers/AlertRules/AlertRules.tsx [hunk 3]
```diff
@@ -155,20 +155,100 @@ class AlertRules extends React.Component<AlertRulesProps, AlertRulesState> {
...
-                  Escalate
-                  <div className={cx('verbal-name')}>{parseEmojis(alertReceiveChannel?.verbal_name || '')}</div>
-                  <Tooltip placement="top" content="Edit name">
+            <div className={cx('integration__heading-container')}>
+              <div className={cx('integration__heading-container-left')}>
+                <Text.Title level={4}>
+                  <div className={cx('integration__heading-text')}>
+                    <div className={cx('verbal-name')}>{parseEmojis(alertReceiveChannel?.verbal_name || '')}</div>
+                    <Tooltip placement="top" content="Edit name">
+                      <span>aaadsa dasd</span>
```
Comment (maskin25) [inline grafana-plugin/src/containers/AlertRules/AlertRules.tsx:164]: `aaadsa dasd` ?

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+}
+
+export const IncidentDropdown: FC<{
+  alert: Alert;
+  onResolve: (e: SyntheticEvent) => Promise<void>;
+  onUnacknowledge: (e: SyntheticEvent) => Promise<void>;
+  onUnresolve: (e: SyntheticEvent) => Promise<void>;
+  onAcknowledge: (e: SyntheticEvent) => Promise<void>;
+  onSilence: (value: number) => Promise<void>;
+  onUnsilence: (event: any) => Promise<void>;
```
Comment (maskin25) [inline grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx:60]: `event: SyntheticEvent` ?

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+import styles from 'pages/incidents/parts/IncidentDropdown.module.scss';
+import { UserActions } from 'utils/authorization';
+
+import { SilenceSelect } from './SilenceSelect';
+
+const cx = cn.bind(styles);
+
+const getIncidentTagColor = (alert: Alert) => {
+  if (alert.status === IncidentStatus.Resolved) {
+    return '#299C46';
```
Comment (maskin25) [inline grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx:20]: it's better to define such color values in `vars.css`

File: grafana-plugin/src/pages/schedules/Schedules.tsx [hunk 6]
```diff
@@ -330,18 +333,24 @@ class SchedulesPage extends React.Component<SchedulesPageProps, SchedulesPageSta
...
-                  <Avatar size="big" src={user.avatar} />
-                  <Text type="secondary"> {user.username}</Text>
-                </div>
-              </PluginLink>
-            );
-          })}
-        </VerticalGroup>
+        <div className="table__email-column">
+          <VerticalGroup>
+            {item.on_call_now.map((user, _index) => {
```
Comment (maskin25) [inline grafana-plugin/src/pages/schedules/Schedules.tsx:338]: can `_index` be omitted here?

File: grafana-plugin/src/style/utils.css [hunk 7]
```diff
@@ -26,6 +26,10 @@
   height: 100%;
 }
 
+.u-display-block {
```
Comment (maskin25) [inline grafana-plugin/src/style/utils.css:29]: what the pattern/convention are you using here? I mean prefixes `u-`

**Review by maskin25 — CHANGES_REQUESTED** [review]

**Comment by maskin25** [comment]
`grafana-plugin/src/containers/AlertRules/AlertRules.tsx` conflict detected

File: grafana-plugin/src/pages/schedules/Schedules.tsx (hunk 6 above)
Comment (teodosii) [reply grafana-plugin/src/pages/schedules/Schedules.tsx:338]: Totally :thumbsup:

File: grafana-plugin/src/style/utils.css (hunk 7 above)
Comment (teodosii) [reply grafana-plugin/src/style/utils.css:29]: It's `u-` from `utility` - it's in the `utils.css` file

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx (hunk 5 above)
Comment (teodosii) [reply grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx:20]: Not sure I understood your point, this is pure JS, how is this related to `vars.css`? We do not import variables from there into JS, or do we?

I have a separate PR where I moved those to `consts.ts` - if you approve that one first I can get the changes here too.

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx (hunk 4 above)
Comment (teodosii) [reply grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx:60]: Done

File: grafana-plugin/src/containers/AlertRules/AlertRules.tsx (hunk 3 above)
Comment (teodosii) [reply grafana-plugin/src/containers/AlertRules/AlertRules.tsx:164]: Oh... that was a stupid mistake. Probably left from testing, I didn't notice it - good catch :thumbsup:

File: grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx (hunk 2 above)
Comment (teodosii) [reply grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx:6]: I'm not sure if you can extend that and add new fields, plus Grafana doesn't seem to import interfaces for props.

Comment (maskin25) [reply grafana-plugin/src/components/MatchMediaTooltip/MatchMediaTooltip.tsx:6]: yep it's possible `type MatchMediaTooltipProps = Pick<TooltipProps, 'placement'|'content'>& {title: string}`, sometimes Grafana exports, sometimes not,  so let's skip this  if Grafana doesn't export in this case

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx (hunk 5 above)
Comment (maskin25) [reply grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx:20]: my point is to save all color values in `vars.css` and use them as `var(--some-color)`, this approach supports theming

**Review by maskin25 — APPROVED** [review]

File: grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx (hunk 5 above)
Comment (teodosii) [reply grafana-plugin/src/pages/incidents/parts/IncidentDropdown.tsx]: Done

**Pull request merged**